lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
+  "env": {}
```

Comment:
Take a look at `decidim.barcelona`'s `app.json`, most of these things are already sorted out:

https://github.com/AjuntamentdeBarcelona/decidim.barcelona/blob/master/app.json